### PR TITLE
Add option to not slide to first index on page resize

### LINKED ISSUE #237: on resize lory bug
Hi lory.
First of all, Thanks good lib.
I found some issue in lory.

I moved to 3rd panel, and then I tried to resize window.
And bug appeared in here, lory panel go to 1st panel.

So, I tried such as code.
But I think it should be default.
Can you give me some advice? 

Thanks for your effort.
```javascript
// persist panel index
elFlick.addEventListener('on.lory.resize', (e) => {
    this.flicking.slideTo(this.currentFlickingIndex);
});
```

### PR DESCRIPTION
**Pull request by riotcku**
fixes #237

Right now it's defaulted to persist the index on resize, can be changed easily.

### CODE REVIEW

File: src/defaults.js [hunk 1]
```diff
@@ -85,5 +85,11 @@ export default {
...
      * @window {object}
      */
-    window: window
+    window: window,
+
+    /**
+     * If false, slides lory to the first slide on window resize.
+     * @persistIndex {boolean}
+     */
+    persistIndex: true
```
Comment (meandmax) [inline src/defaults.js]: Sorry for the late answer: I would say to not have breaking changes, default should be false. And I would name this `rewindOnResize` to be consistent

**Comment by riotcku** [comment]
@meandmax addressed feedback!

**Comment by meandmax** [comment]
Hey @riotcku,

Thanks for the adjustment, I think I wrote my feedback a little confusing. If we call it rewindOnResize I think the default should be `true` because this is the current default, it rewinds on resize. And then please remove the `!` from here:

https://github.com/meandmax/lory/pull/312/commits/c6afac5cf4c264856cea42afce3b66ee0e716ade#diff-b9ae7b7c349009633969d97d3d11587fR282

Then the semantic and the syntax is correct, sorry for not communicating this right.

And could you just change your commit msg to something like `feat: add option to turn rewindOnResize off` otherwise semantic-release does not know what to do ...

see here:
https://github.com/semantic-release/semantic-release
and:
https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit

Thanks for contributing!
Max

**Comment by meandmax** [comment]
obsolete by #401

**Pull request closed without merging**